learning-team/2024/2024-06-17.md

```diff
@@ -0,0 +1,52 @@
+# 2024-06-17
+
+Note Taker: Chris Manson
+Time Keeper: Anne-Greeth Schot-van Herwijnen
+
+## Attendees
+
+Add yourself to the list if you attend and check the box!
+
+### Core Team
+
+- [x] Chris Manson (CM)
+- [x] Jared Galanis (JG)
+- [ ] Jen Weber (JW)
+- [ ] Ricardo Mendes (RM)
+- [ ] Robert Wagner (RW)
+- [x] Anne-Greeth Schot-van Herwijnen (AG)
+- [x] Ignace Maes (IM)
+
+## Topics
+
+### Feature Flag / Toggle <@jaredgalanis>
+
+- https://github.com/ember-learn/guidemaker-ember-template/pull/191
+
+- JG: discussion about feature flags
+- IM: basic implementaion is imporant
+- then we can start using it for other things like template tags
+- JS vs TS is another discussion
+- should we have switches per code block or globally
+- CM: I think we talked about needing surrounding blurbs
+- that way we shouldn't be 
+- IM: we need to unblock polaris docs, we can get people to start contributing
+- CM: assuming the tech part is done, how do we deal with multiple toggles at the same time? polaris and typescript
+- do we need 4 versions of the docs?
+- CM: we need to be clear about the point of the feature flag
+- it is intended to be a mechanism to preview the new guides
+- (discussion about the feature flag)
+- (description of CM's idea of how the toggle will exactly work)
+- Action Item: CM to write down the exact plan so get everyone in the core teams can agree on the plan
+- (discussion about sub-flags)
+
+### Deprecations PR from Katie/Nullvox <@jaredgalanis>
+
+- https://github.com/ember-learn/deprecation-app/pull/1383
+- CM: I don't think this is highest priority, I don't think we should complicate the data structure of the deprecations
```

Comment:
is there a blocker?

I'm confused about "highest priority" since the work is already done :sweat_smile:

Reply:
Rogerroger.gif